### ISSUE TITLE
Add an option to translate an existing subtitle, like on YouTube. Completely or random sparse words to learn. Possibility to choose principal translation + learning translation, so any language can be used to learn any language from any language too

### ISSUE THREAD

**Issue by bphd** [osmansonsuz/stremio-subtitle-addon#1]

**Comment by osmansonsuz** [comment]
Thanks for the idea. I will work on it.

**Comment by bphd** [comment]
> @[osmansonsuz](https://github.com/osmansonsuz) osmansonsuz closed this as [completed](https://github.com/osmansonsuz/stremio-subtitle-addon/issues?q=is%3Aissue+is%3Aclosed+archived%3Afalse+reason%3Acompleted) [Aug 19, 2023](https://github.com/osmansonsuz/stremio-subtitle-addon/issues/1#event-10136451704)

Already!? Subhan Allah, so fast. Give the link of the add-on that does that

**Comment by osmansonsuz** [comment]
> > @[osmansonsuz](https://github.com/osmansonsuz) osmansonsuz closed this as [completed](https://github.com/osmansonsuz/stremio-subtitle-addon/issues?q=is%3Aissue+is%3Aclosed+archived%3Afalse+reason%3Acompleted) [Aug 19, 2023](https://github.com/osmansonsuz/stremio-subtitle-addon/issues/1#event-10136451704)
> 
> Already!? Gosh, so fast. Give the link of the add-on that does that

No. I closed it because i read it. So.. thats not the way it works i guess. Reopening the issue.

**Comment by bphd** [comment]
Yeah, not the way... The way it is now you assign you create a task to do that, you assign it to a milestone, a future version of the add-on, and naturally, you communicate about the release cycle, so we know when to expect the new function. And don't forget to constantly push new code to complete the task and update its status and so on

Then waiting it is
